tsql/tsql.g4

```diff
@@ -586,6 +588,14 @@ use_statement
     : USE database=id ';'?
     ;
 
+dbcc_clause
+    : DBCC name=simple_id ('(' expression_list ')')? (WITH dbcc_options)? ';'?
```

Comment:
This is a simplified version, DBCC clause is quite versatile in MSDN https://msdn.microsoft.com/en-us/library/ms176064.aspx, there could be more simple_id added to the clause in future.